scripts/release-binary.sh

```diff
@@ -44,14 +44,16 @@ function usage() {
   echo "$0
     -d  The bucket name to store proxy binary (optional).
     -i  Skip Ubuntu Xenial check. DO NOT USE THIS FOR RELEASED BINARIES.
-        Cannot be used together with -d option."
+        Cannot be used together with -d option.
+    -s  Skip pushing envoy docker image."
   exit 1
 }
 
-while getopts d:i arg ; do
+while getopts d:i:s arg ; do
   case "${arg}" in
     d) DST="${OPTARG}";;
     i) CHECK=0;;
+    s) PUSH_DOCKER_IMAGE="";;
```

Comment:
You might want to switch this to `0`/`1` to match `CHECK`.

Reply:
Done